.env.example

```diff
@@ -58,4 +58,8 @@ SESSION_RESET_THRESHOLD_TEST=2000
 
 # Threshold for sending out a new message for a new alert during a session.
 SUBSEQUENT_ALERT_MESSAGE_THRESHOLD=120
-SUBSEQUENT_ALERT_MESSAGE_THRESHOLD_TEST=1000
\ No newline at end of file
+SUBSEQUENT_ALERT_MESSAGE_THRESHOLD_TEST=1000
+
+# How large a window we use to calculate moving average of radar values
+RADAR_WINDOW_SIZE_SECONDS=15
```

Comment:
Could you please pre-add these values to the staging and prod secrets or add a subtask to do it in the deploy task?